scrapper.py

```diff
@@ -25,39 +40,154 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        """
+        get link to the article
+        """
+        for article_link in article_bs.find_all("a", class_="article__title"):
+            self.urls.append(DOMAIN + article_link["href"])
```

Comment:
I do not see you controlling the number of collected articles

Reply:
I added a counter to this class